src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,36 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+export const PostInfo = ({ post }) => {
+  const { title, body, user, comments } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+
+        <p>
+          {' Posted by '}
```

Comment:
Instead of using the string ' Posted by ' directly, it would be better to put it in a span or p tag for better styling and structuring.